src/url.zig

```diff
@@ -1024,7 +1029,10 @@ pub const FormData = struct {
                 .form = form,
             };
 
-            try forEachMultipartEntry(input, boundary, *Wrapper, &wrap, Wrapper.onEntry);
+            forEachMultipartEntry(input, boundary, *Wrapper, &wrap, Wrapper.onEntry) catch {
```

Comment:
This removes the original error message. We can preserve the error by returning it

```suggestion
            forEachMultipartEntry(input, boundary, *Wrapper, &wrap, Wrapper.onEntry) catch |err| {
            
```

Reply:
fixed on https://github.com/oven-sh/bun/pull/2803/commits/92e9530f0a2e5cc39e6e3864edc3fa32049fbd7e#diff-e84dfe2cc8a38ad62da56117bfcad3d57569be91619624b3383dded4c5aae319